test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaAttributeTests.java

```diff
@@ -154,11 +154,24 @@ static public void testStaticNullRestrictedFieldWhereImplicitCreationHasNoDefaul
 		c.newInstance();
 	}
 
+	static public Class<?> TestPutFieldNullToNullRestrictedFieldClass = null;
```

Comment:
nit: I'd recommend starting these variable names with a lowercase to follow Java variable naming conventions of lower camel case.

Reply:
Addressed in 32f6437